packages/core/src/internal/motion/motion.stories.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/button/register.js';
+import '@cds/core/internal-components/overlay/register.js';
+import '@cds/core/accordion/register.js';
+// import { ClarityMotion } from '@cds/core/internal/motion/index.js';
+// import { html } from 'lit-html';
+// import customElements from '../../dist/core/custom-elements.json';
```

Comment:
This file may not be needed. Do you think I should remove it, @coryrylan  ?

Reply:
Yeah if theres no motion specific stories then it can be removed. It might be breaking the other stories as well since it has a title of `title: 'Stories/Typography'`